app/services/perform_test_model_version_run_jobs.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class PerformTestModelVersionRunJobs
+  attr_reader :test_run_id, :test_model_version_run_id
+
+  def initialize(test_run_id, test_model_version_run_id)
```

Comment:
I think we can use model for test_run in initializer

Reply:
pls have a look at https://github.com/aifoundry-org/llamagator/pull/55/commits/0fb0092cccfa455a72045220e307a17de3a4bf55